indico/modules/events/contributions/templates/display/contribution_display.html

```diff
@@ -145,7 +145,7 @@
         {% call message_box('warning') -%}
             {% trans %}
                 The contributions of this event have not been published yet. You can only access
-                this page because you are an event manager.
+                this page because you are an event manager or you are associated with this contribution.
```

Comment:
I don't think this message should be shown in this case at all - it's just going to confuse users...

Reply:
So, only show it to managers and the submitters see it as a regurlar page?

Reply:
I think that's the most logical solution: For a manager it's important to know because they may otherwise think the link can be shared with others or forget to publish in the end. But for a user it's pretty much expected that they can see their own contribution even if the full list of contributions hasn't been released yet.